assets/app/views/_alerts.html

```diff
@@ -1,20 +1,32 @@
 <div ng-repeat="alert in alerts">
   <div ng-switch="alert.type">
-    <div ng-switch-when="error" class="alert alert-danger">
+    <div ng-switch-when="error" class="alert alert-danger" ng-class="{'alert-dismissable': alert.dismissable}">
+      <button ng-if="alert.dismissable" type="button" class="close" data-dismiss="alert" aria-hidden="true">
```

Comment:
Does this work properly with multiple alerts or when an alert is dismissed, then another alert with the same type is displayed?